core/object/script_language.h

```diff
@@ -53,7 +53,9 @@ class ScriptServer {
 	static int _language_count;
 	static bool scripting_enabled;
 	static bool reload_scripts_on_save;
-	static SafeFlag languages_finished; // Used until GH-76581 is fixed properly.
+	static bool languages_finished; // Used until GH-76581 is fixed properly.
+	static bool language_initialized;
```

Comment:
```suggestion
	static bool language_initialized; // Used until GH-76581 is fixed properly.
```
And remove from above